src/definitions/chakra/theme/components/button.ts

```diff
@@ -0,0 +1,24 @@
+const Button = {
+    // The styles all button have in common
+    baseStyle: {},
+    // Two sizes: sm and md
+    sizes: {},
+    // Two variants: outline and solid
+    variants: {
+        default: {
+            bg: "background.dark",
+            color: "white",
+            _hover: { bg: "gray.700" },
+            _active: { bg: "gray.600" },
+        },
+        outline: {
+            colorScheme: "black",
```

Comment:
![image](https://user-images.githubusercontent.com/42286268/125714715-e58fa034-b1d3-4ebe-b156-69f81d8fea41.png)
Is "black" is a valid value for colorScheme?

Reply:
This will be fixed in an upcoming PR. `outline` will be changed to `outlineBlack` to avoid conflicting with defaults.